nomenclature/processor/required_data.py

```diff
@@ -16,6 +16,7 @@
 
 from nomenclature.definition import DataStructureDefinition
 from nomenclature.error import ErrorCollector
+from nomenclature.processor.iamc import IamcDataFilter
```

Comment:
I'd say for now at least you can take this one out since `IamcDataFilter` is not actually used yet:
```suggestion
```